### DISCUSSION TITLE
dbt Cloud Integration problem

### DISCUSSION THREAD

**Discussion by gurbuzertunga** [airbytehq/airbyte#39426] [Connector Questions]
I am trying to add a new service token for dbt Cloud Integration but your system keeps throwing errors.http.internalServerError.

I created the dbt account and generated the service token. when pasted, airbyte cloud integrations throws the error.

here is a screenshot as well

![Screenshot from 2024-06-10 18-41-19](https://github.com/airbytehq/airbyte/assets/65021553/7ffe7e1f-2ad6-4b16-8e7a-eced5866d8fd)

**Comment by teallarson** [comment]
I agree this is a less than helpful error message!  We have an update coming soon to this integration that should provide more helpful error handling.

Two thoughts: 
- Is your dbt account is hosted outside the North America region? ([dbt regions docs](https://docs.getdbt.com/docs/cloud/about-cloud/access-regions-ip-addresses))  That is a limitation of our current integration.  The update we've been working on is going through some final testing before release, but should open up additional regions!
- Is your dbt Cloud account is on the free tier? The integration is only able to support team or enterprise tier accounts.

**Reply by gurbuzertunga** [reply]
> I agree this is a less than helpful error message! We have an update coming soon to this integration that should provide more helpful error handling.
> 
> Two thoughts:
> 
> * Is your dbt account is hosted outside the North America region? ([dbt regions docs](https://docs.getdbt.com/docs/cloud/about-cloud/access-regions-ip-addresses))  That is a limitation of our current integration.  The update we've been working on is going through some final testing before release, but should open up additional regions!
> * Is your dbt Cloud account is on the free tier? The integration is only able to support team or enterprise tier accounts.

Hi, I Checked both points and yes, Account is in North America Region and it's a Team account trial. Error still persists tho.

**Comment by marcosmarxm** [comment]
Duplicated of https://github.com/airbytehq/airbyte/issues/39428 Let's continue the discussion in the issue.
